src/test/scala/uk/gov/hmrc/test/ui/pages/section4/PreviousDocumentListPage.scala

```diff
@@ -44,4 +44,6 @@ object PreviousDocumentListPage extends BasePage {
   // Yes => fillPage(yes)
 
   override def fillPage(values: String*): Unit = selectYesOrNoRadio(values(yesNo))
+
+  def removePreviousDoc(index: Int): Unit = clickByCssSelector(s"#previous-document-$index-remove_button>a")
```

Comment:
Use findChildByTag

Reply:
Hello siva, I do not see method  findChildByTag in driver helper . The only method i see is findChildrenByTag and it is a Seq[Webelements]. I donot need a sequence of webelements. Also i need to create a separate method for click again if i am using findChildrenByTag. So i think its better to use clickByCssSelector.

Reply:
def removeAuthCode(index: Int): Unit =
    findChildrenByTag(findElementById(s"holder-table-row$index-remove_button"), "a")

Reply:
you can use like this Dhanist